Casks/boinc.rb

```diff
@@ -3,11 +3,15 @@
   sha256 "9a0eabad4b6780f56543c5634bfcef46728bd77f95f3d5e46cc35a0b6bdbdce1"
 
   url "https://boinc.berkeley.edu/dl/boinc_#{version}_macOSX_universal.zip"
-  appcast "https://www.macupdater.net/cgi-bin/extract_text/download_with_useragent.cgi?url=https://boinc.berkeley.edu/download.php&user_agent=Mac"
   name "Berkeley Open Infrastructure for Network Computing"
   name "BOINC"
   homepage "https://boinc.berkeley.edu/"
 
+  livecheck do
+    url "https://boinc.berkeley.edu/download.php"
+    regex(/boinc_(.*)_macOSX_universal\.zip/i)
```

Comment:
```suggestion
    regex(%r{href=.*?/dl/boinc[._-]v?(\d+(?:\.\d+)+)_macOSX_universal\.zip}i)
```

With this version of the regex, it's necessary to include `/dl/` or else it will return a version like `berkeley.edu/dl/boinc_7.16.14`.